src/main/scala/com/yotpo/metorikku/configuration/job/output/Hudi.scala

```diff
@@ -6,6 +6,7 @@ case class Hudi(dir: String,
                 operation: Option[String],
                 storageType: Option[String],
                 maxVersions: Option[String],
+                recordSizeEstimate: Option[Int],
```

Comment:
Let's all our parameters from change-data-capture project to Hudi.scala as proper configs, WDYT?

Reply:
it was pushed by mistake

Reply:
Still, I like it
If there are parameters we find interesting let's add them!